swift/apple/FirezoneKit/Sources/FirezoneKit/Features/MainView.swift

```diff
@@ -15,70 +15,47 @@ import SwiftUI
   final class MainViewModel: ObservableObject {
     private let logger: AppLogger
     private var cancellables: Set<AnyCancellable> = []
+    let tunnelStore: TunnelStore
 
-    let appStore: AppStore
     @Dependency(\.mainQueue) private var mainQueue
 
-    @Published var loginStatus: AuthStore.LoginStatus = .uninitialized
-    @Published var tunnelStatus: NEVPNStatus = .invalid
-    @Published var orderedResources: [DisplayableResources.Resource] = []
+    @Published private(set) var resources: [Resource]?
 
-    init(appStore: AppStore) {
-      self.appStore = appStore
-      self.logger = appStore.logger
+    init(tunnelStore: TunnelStore, logger: AppLogger) {
+      self.tunnelStore = tunnelStore
+      self.logger = logger
       setupObservers()
     }
 
     private func setupObservers() {
-      appStore.authStore.$loginStatus
-        .receive(on: mainQueue)
-        .sink { [weak self] loginStatus in
-          self?.loginStatus = loginStatus
-        }
-        .store(in: &cancellables)
-
-      appStore.tunnelStore.$status
+      tunnelStore.$status
         .receive(on: mainQueue)
         .sink { [weak self] status in
-          self?.tunnelStatus = status
+          guard let self = self else { return }
           if status == .connected {
-            self?.appStore.tunnelStore.beginUpdatingResources()
+            self.tunnelStore.beginUpdatingResources()
           } else {
-            self?.appStore.tunnelStore.endUpdatingResources()
+            self.tunnelStore.endUpdatingResources()
           }
         }
         .store(in: &cancellables)
 
-      appStore.tunnelStore.$resources
+      tunnelStore.$resourceListJSON
         .receive(on: mainQueue)
-        .sink { [weak self] resources in
-          guard let self = self else { return }
-          self.orderedResources = resources.orderedResources.map {
```

Comment:
They are, but this was named this because there were two types in the Apple client `resources` and `orderedResources`, before connlib sorted them.

The `resources` was removed, leaving `orderedResources`, which to me implies we ordered them somewhere, but we don't. We don't actually care what order they're in anymore. We populate the list from the initial `resourceListJSON` and then from there we update the items in-place using their `address` and `name`.